tests/nn/test_module.py

```diff
@@ -532,7 +643,9 @@ def randomize(model):
     assert_identical(actual, expected)
 
 
-def test_torch_serialize_attributes():
+@pytest.mark.parametrize("local_params", [True, False])
+def test_torch_serialize_attributes(local_params):
+    pyro.settings.set(module_local_params=local_params)
```

Comment:
Could we use a `pyro.settings.context` here so as not to leak across other tests?

Reply:
Done